### PR TITLE
Feature/cplp 1190/enable marketplace navigation

### PR DESCRIPTION
**Pull request by nidhigarg-bmw**
I also made the ownUser name changes in this branch accidently

### CODE REVIEW

File: cx-portal-shared-components/src/components/basic/Menu/MenuItem.tsx [hunk 1]
```diff
@@ -66,6 +69,7 @@ export const MenuItem = ({
         {...props}
       >
         {title}
+        {hint && <Chip label={hint} type="plain" sx={{ height: 'auto' }} />}
```
Comment (oyo) [inline cx-portal-shared-components/src/components/basic/Menu/MenuItem.tsx:72]: now we have line breaks in the menu items - this is not nice.
image
Three ways to solve this:
- make menus wider
- make the hint smaller
- make the menu item text smaller

@jjeroch - what is your preferred solution?

**Review by oyo — COMMENTED** [review]
what do we do about those line breaks?

File: cx-portal-shared-components/src/components/basic/Menu/MenuItem.tsx (hunk 1 above)
Comment (jjeroch) [reply cx-portal-shared-components/src/components/basic/Menu/MenuItem.tsx:72]: @oyo  I suggest we extend the menu. We are not really limited to the size, I think a wider size based on the content is a good solution. Have you had a look at this, Does it look awkward? I dont think so, or?

Comment (oyo) [reply cx-portal-shared-components/src/components/basic/Menu/MenuItem.tsx:72]: @jjeroch - i checked out the branch and tested locally. Didn't you see the attached screen shot? I think it doesn't look very nice.

Comment (jjeroch) [reply cx-portal-shared-components/src/components/basic/Menu/MenuItem.tsx:72]: @oyo are we talking about the same? I was talking about extending the wide.
But the screenshot is two lines.

Comment (oyo) [reply cx-portal-shared-components/src/components/basic/Menu/MenuItem.tsx:72]: @jjeroch this is how it looks wider:
image

but a little bit awkward for short menu names
image

Comment (jjeroch) [reply cx-portal-shared-components/src/components/basic/Menu/MenuItem.tsx:72]: @oyo i would suggest we go for a bigger wide when needed. Means the sub-menu keeps the current used size when no tag is added. As soon as a tag is in, the box is increasing in its wide

Comment (oyo) [reply cx-portal-shared-components/src/components/basic/Menu/MenuItem.tsx:72]: how about this - keeping the original width but minmal label
image

Comment (jjeroch) [reply cx-portal-shared-components/src/components/basic/Menu/MenuItem.tsx:72]: but thats too small, we need to ensure that it is readable. No issue with a smaller label, but thats too small

Comment (oyo) [reply cx-portal-shared-components/src/components/basic/Menu/MenuItem.tsx:72]: @jjeroch we found this looks quite good. what do you say?
image

Comment (jjeroch) [reply cx-portal-shared-components/src/components/basic/Menu/MenuItem.tsx]: sorry missed your new comment. Lets take this! But would you mind to go for purple ;) :D

Comment (oyo) [reply cx-portal-shared-components/src/components/basic/Menu/MenuItem.tsx]: changed to a nice purple - @nidhigarg-bmw can you pull and finish the other remarks like moving the i18n texts from footer to main?

Comment (jjeroch) [reply cx-portal-shared-components/src/components/basic/Menu/MenuItem.tsx]: :D that is ping :D but okay, I interpret that as artistic freedom ;)

Comment (nidhigarg-bmw) [reply cx-portal-shared-components/src/components/basic/Menu/MenuItem.tsx]: @oyo I made the changes. Please recheck

**Review by oyo — APPROVED** [review]

**Pull request merged**